docs/interfaces/auth-service.txt

```diff
@@ -1,20 +1,45 @@
 Interface Token {
-    payload: {
-        // The lifetime of the Token. The date in unix seconds of the expiration.
-        expiresAt: date,
-        // The corresponding userId of the requesting client.
-        userId: number,
-        // The id of the current session.
-        sessionId: string
-    },
-    signature: string
+    // The lifetime of the Token. The date in unix seconds of the expiration.
+    expiresAt: date,
+    // The corresponding userId of the requesting client.
+    userId: number,
+    // The id of the current session.
+    sessionId: string,
+    // Date of creation of a token.
+    iat: number
 }
 
 Interface Cookie {
-    // The id for the session corresponding to the client, who has signed in.
+    // The id for the session corresponding to the user's client.
     sessionId: string,
-    // A signature created from the server.
-    signature: string
+}
+
+/**
+* Describes an http-response, which is sent back to any requesting service.
+*/
+Interface Response <V, T> {
+    // Optional headers, which are set in an http-response
+    [ Headers?: V ]
```

Comment:
Do you mean `headers?: {[name: string]: string}?` or `[header: string]: V` -> Are ehaders in an extra object or just extra keys in the response?